lightwood/mixers/nn.py

```diff
@@ -600,10 +603,10 @@ def _iter_fit(self, ds, subset_id=None, max_epochs=120000):
                         target_loss = target_loss.tolist()
                         if type(target_loss[0]) == type([]):
                             target_loss = [np.mean(x) for x in target_loss]
-                        for i, value in enumerate(target_loss):
-                            if len(unreduced_losses) <= i:
+                        for j, value in enumerate(target_loss):
```

Comment:
why the change from i to j? Is it cosmetic?

Reply:
Just to be sure. There is an `i` variable already being used a little farther up. The closure in the for loop should mean it's not an issue, but it's probably safer to change it so that one doesn't make mistakes later on.